plugins/SegmentEditor/templates/_unprocessedSegmentMessage.twig

```diff
@@ -0,0 +1,10 @@
+<p>
+    This report has no data, because the Segment you requested <strong>({{ segmentName }})</strong> has not yet been processed by the system.
+    Data for this Segment will become available in a few hours when processing completes.
+</p>
+<p>&nbsp;</p>
+<p>
+    <a href="{{ visitorLogLink }}" target="_blank">In the Visitor Log</a>, you can test whether your segment will match your users
+    correctly by applying it there. When applied, you can see immediately which visits and actions were matched by your segment.
+    This can help you confirm your Segment matches the users and actions you expect it to.
+</p>
```

Comment:
Any reason for not adding this translatable?

Reply:
Oh, right, forgot